src/components/DeliverableView/QuestionsDeliverableCard.tsx

```diff
@@ -111,7 +144,12 @@ const QuestionsDeliverableCard = (props: EditProps): JSX.Element | null => {
           }}
         >
           {filteredVariablesWithValues.map((variableWithValues: VariableWithValues, index: number) =>
-            variableDependencyMet(variableWithValues, filteredVariablesWithValues) ? (
+            variableDependencyMet(
+              variableWithValues,
+              specificVariablesWithValues
```

Comment:
This is really minor, but is it possible to move this to another `useMemo` under `filteredVariablesWithValues`, mostly just to be consistent with where the other variablesWithValues are defined? Maybe can be named `dependentVariablesWithValues`